src/store/initialState.js

```diff
@@ -10,6 +10,10 @@ const initialState = {
       error: ''
     },
     isAuthenticated: false
+  },
+  articleCategory: {
+    category: null,
+    categories: [],
   }
```

Comment:
I think the whole idea of the `initialState.js` file is to give a clear view of what the app state looks like when loaded. In that light you can make the values in your initial state more descriptive of the value it would finally hold. For instance `category: null` does not let one know what the shape of `category` will be like later on.

Also you have `categories: []` but in the `reducers/category.js` file you set it to be an object with shape 
```
      const categories = {
        isLoading: false,
        isError: false,
        data: action.payload
      };
```

I think you can declare this shape in the initial state so anyone looking through the code knows what the shape/interface of `categories` looks like. Same thing goes for the `category` state.

Reply:
Okay. I will include these in the initial state